lib/app.rb

```diff
@@ -0,0 +1,86 @@
+require_relative "../lib/item_repository.rb"
```

Comment:
This is impressively well-organised code. Very readable. It communicates it's function to other devs intrinsically.